webapp/components/_new/features/SearchResults/SearchResults.vue

```diff
@@ -0,0 +1,338 @@
+<template>
+  <div id="search-results" class="search-results">
+    <div>
+      <ds-text class="total-search-results">
+        {{ $t('search.for') }} "
+        <strong>{{ search }}</strong>
+        "
+        <br />
+        <strong>{{ searchCount }}</strong>
+        {{ $t('search.results', {}, searchCount) }}
+      </ds-text>
+    </div>
+
+    <tab-navigation :tabs="tabOptions" :activeTab="activeTab" @switchTab="switchTab" />
+    <section
+      :class="['results', activeTab === 'User' && '--user', !activeResourceCount > 0 && '--empty']"
+    >
+      <hc-empty
+        v-if="!activeResourceCount || searchCount === 0"
+        icon="tasks"
+        :message="$t('search.no-results', { search })"
+      />
+      <template>
+        <pagination-buttons
+          v-if="activeResourceCount > pageSize"
+          :hasNext="hasNext"
+          :showPageCounter="true"
+          :hasPrevious="hasPrevious"
+          :activePage="activePage"
+          :activeResourceCount="activeResourceCount"
+          :key="'Top'"
+          :pageSize="pageSize"
+          @back="previousResults"
+          @next="nextResults"
+        />
+        <masonry-grid v-if="activeTab === 'Post'">
+          <masonry-grid-item v-for="resource in activeResources" :key="resource.key">
+            <post-teaser :post="resource" />
+          </masonry-grid-item>
+        </masonry-grid>
+
+        <ul v-if="activeTab === 'User'" class="user-list">
+          <li v-for="resource in activeResources" :key="resource.key" class="item">
+            <base-card :wideContent="true">
+              <user-teaser :user="resource" />
+            </base-card>
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-09-23T06:37:30Z" title="Wednesday, September 23rd 2020, 8:37:30 am +02:00">Sep 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,328 @@
+<template>
+  <div id="search-results" class="search-results">
+    <div>
+      <ds-section>
+        <ds-text class="total-search-results">
+          <strong>{{ searchCount }}</strong>
+          {{ $t('search.results', {}, searchCount) }}
+        </ds-text>
+      </ds-section>
+    </div>
+
+    <tab-navigation :tabs="tabOptions" :activeTab="activeTab" @switchTab="switchTab" />
+    <section
+      :class="['results', activeTab === 'User' && '--user', !activeResourceCount > 0 && '--empty']"
+    >
+      <hc-empty
+        v-if="!activeResourceCount || searchCount === 0"
+        icon="tasks"
+        :message="$t('search.no-results', { search })"
+      />
+      <template>
+        <pagination-buttons
+          v-if="activeResourceCount > pageSize"
+          :hasMoreResults="hasMoreResults"
+          :hasPreviousResult="hasPreviousResult"
+          :activePage="activePage"
+          :activeResourceCount="activeResourceCount"
+          @back="previousResults"
+          @next="nextResults"
+          :key="'Top'"
+          :pageSize="pageSize"
+        />
+        <masonry-grid v-if="activeTab === 'Post'">
+          <masonry-grid-item v-for="resource in activeResources" :key="resource.key">
+            <post-teaser :post="resource" />
+          </masonry-grid-item>
+        </masonry-grid>
+
+        <ul v-if="activeTab === 'User'" class="user-list">
+          <li v-for="resource in activeResources" :key="resource.key" class="item">
+            <base-card :wideContent="true">
+              <user-teaser :user="resource" />
+            </base-card>
+          </li>
+        </ul>
+        <ul v-if="activeTab === 'Hashtag'" class="hashtag-list">
```


Empty line between this like above? ☝🏼